packages/engine/Specs/Scene/MultifrustumSpec.js

```diff
@@ -237,7 +237,7 @@ describe(
           vs += "in vec4 position;";
           vs += "void main()";
           vs += "{";
-          vs += "    gl_Position = czm_modelViewProjection * position;";
+          vs += "    vec4 positionEC = czm_modelView * positiong; l_Position = czm_projection * positionEC;";
```

Comment:
There is a typo here: this line should be split across 2 lines.

We generally want to use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) instead of string concatenation. See https://github.com/CesiumGS/cesium/pull/10067. This typo is a good example of why: manually adding up strings is hard to read and easy to make a small mistake.

Can you update this section to use a template literal?